.distignore

```diff
@@ -1,4 +1,5 @@
 # Directories
+/.wordpress-org/
```

Comment:
Looks like action-wordpress-plugin-deploy defaults to `.wordpress-org` being the default name for the `assets` directory so there is no additional config need for the GitHub Action.